docker-compose.deploy.yml

```diff
@@ -370,15 +390,18 @@ services:
     deploy:
       labels:
         - 'traefik.enable=true'
-        - 'traefik.frontend.rule=Host: register.{{hostname}},{{hostname}}'
-        - 'traefik.frontend.redirect.regex=^https?://{{hostname}}/(.*)'
-        - 'traefik.frontend.redirect.replacement=https://register.{{hostname}}/$${1}'
```

Comment:
I think it was a nice UX to redirect the user to the `register` subdomain if none were specified, don't you think?